<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="11" column="1">
     <widget class="QSlider" name="sliderFrequ">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="edtURLA2C"/>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Parameters</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="edtParamC2A"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Serial Port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="selPort"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextEdit" name="txtLog"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Main URL</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Enable Arduino to Cloud</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Update frequency [1 to 60 s]</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="edtParamA2C"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Parameters</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Http rest request for actuators update (Cloud to Arduino)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="edtURLC2A"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Http rest request for Server Update (Arduino to Cloud)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Main URL</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="chkEnA2C">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Enable Cloud to Arduino</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QCheckBox" name="chkEnC2A">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
